src/Seq.App.Slack/SlackReactor.cs

```diff
@@ -59,10 +55,23 @@ public class SlackReactor : Reactor, ISubscribeTo<LogEventData>
             IsOptional = true)]
         public string IconUrl { get; set; }
 
+        [SeqAppSetting(
+            DisplayName = "Proxy Server",
+            HelpText = "Proxy server to be used when making HTTPS request to slack api, uses default credentials",
+            IsOptional = true)]
+        public string ProxyServer { get; set; }
+
+        [SeqAppSetting(
+            DisplayName = "Dictionary serialiser string limit",
+            IsOptional = true,
+            HelpText = "If a property is a Dictionary, it's serialised as JSON and will be truncated if the resulting string is longer than this number")]
```

Comment:
Could this just apply to all property values? I can see arrays suffering from similar issues.

Reply:
Makes sense, it applies to everything now. Would it make sense to set it to a sensible limit (500?) by default? I've just left it as null by default

Reply:
Awesome, thanks for all the updates :+1: 

I think defaulting to `null` is probably the most conservative choice for a start, perhaps we see how this gets used, and consider another default down the track.